.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: php
+
+php:
+    - 5.4
```

Comment:
given https://github.com/ezsystems/ezplatform/blob/master/composer.json#L20, I guess we should test in 5.5, 5.6 and 7, no ?